build/config.yaml

```diff
@@ -5,7 +5,6 @@ sidecars:
     X_CSI_SPEC_VERSION: v1.0
     external-attacher: quay.io/k8scsi/csi-attacher:v1.1.1
     external-provisioner: quay.io/k8scsi/csi-provisioner:v1.1.0
-    cluster-driver-registrar: quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.1
```

Comment:
Are you sure we should remove this?  CSI v1.0 works on K8s v1.13, which still needs the driver registrar...

Reply:
Not sure - it's the default, one can always set X_EMBER_OPERATOR_CLUSTER=k8s-1.13 if needed? This brings up the question how long do we support older cluster versions by default.

Maybe we need an internal check on the "real" cluster version as well, and raise a warning if one of the needed sidecars is not defined?

Reply:
We should remove it like you intended, that way newer versions of k8s/openshift won't be deploying it and not creating the CSIDriver.